scripts/reinstall.sh

```diff
@@ -7,7 +7,6 @@ set -e
 set -x
 
 rm -rf node_modules
```

Comment:
I believe removing `node_modules` in the new default case (`npm ci`) doesn't do anything but prevent the fast path from being hit where Node can diff `package-lock.json` and the local `.package-lock.json` in `node_modules`. https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json#hidden-lockfiles

`npm ci` should end up with the exact same `node_modules` no matter if you delete it or not, because at the hash of the `node_modules` has to match the hash in `package-lock.json` (at least before `postinstall` step)

What is the desired behavior from `npm ci` after deleting the `node_modules`?

Reply:
Hm, interesting.  I'll read more.  Maybe I've confounded things, but I feel like I've landed in uncertain state from this before.

If the contract of `npm ci` is "your directory will be exactly as it is supposed to be, including deleting unwanted files/folders/versions", then yeah, this pr is unnecessary

Reply:
Yeah I thought the same thing but seems like `npm install` and `npm ci` have been pruning stuff for a while now. We should test it to be sure though, not 100% convinced that the theoretical matches the practical either...

Reply:
Ah actually I know why this may work... 

`.turbo` and other `.x` dirs in `node_modules` are not wiped by `npm ci`...
We could then perhaps be more surgical, but maybe best just to leave it as is.

Reply:
I debated adding `rm -rf .next` and `rm -rf .turbo` to each of the subdirectories.

That's more of a "clean" than "reinstall" though.  Maybe another script for another day.